packages/shared/src/utils/platform/platform.js

```diff
@@ -11,6 +11,10 @@ export const platform_name = Object.freeze({
     SmartTrader: 'SmartTrader',
 });
 
+export const isDashboard = () =>
+    /^\/dashboard/.test(window.location.pathname) ||
+    (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === 'dashboard');
+
```

Comment:
we are removing `dashboard` path in the below PR
https://github.com/binary-com/deriv-app/pull/2622/files#diff-707e536d86222f1c30f7c569c7a9b66dfecc30fb46ba0bda4ebd5a51bdef520aR53

This function may not work as expected. Can you check this please? 🙏

Reply:
@balakrishna-binary thanks, wasn't aware of this.